### PR TITLE
UX: fixes for new login/signup modals

### PR DESCRIPTION
**Pull request by awesomerobot**

### CODE REVIEW

File: .discourse-compatibility [hunk 1]
```diff
@@ -1 +1 @@
-2.6.0.beta3: 49f2f8d75cfd449b3ae65092fb43d7e6716ca94f
```
Comment (JefryMW23) [inline .discourse-compatibility:1]: Jj

Comment (JefryMW23) [inline .discourse-compatibility:1]: Dksks

File: .discourse-compatibility [hunk 2]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+2.7.0.beta2: 7cb06ad2f9c7db7722f7bd2b91f4e1754ab01260
```
Comment (JefryMW23) [inline .discourse-compatibility:1]: Hdjd

**Review by JefryMW23 — COMMENTED** [review]
@JefryMW23 

>

File: .discourse-compatibility (hunk 2 above)
Comment (JefryMW23) [inline .discourse-compatibility:1]:
```suggestion
2.7.0.beta2: 7cb06ad2f9c7db7722f7bd2b91f4e1754ab01260
```

**Comment by CvX** [comment]
@awesomerobot since the spam bot dug up this PR 😛 is it still relevant? (and should be reviewed? 🙂)

**Pull request closed without merging**